redisvl/redis/utils.py

```diff
@@ -20,12 +20,12 @@ def convert_bytes(data: Any) -> Any:
             return data.decode("utf-8")
         except:
             return data
-    if isinstance(data, dict):
```

Comment:
just curious why `elif` given the return path?

Reply:
Changed it back to short-circuit ifs